src/routes.js

```diff
@@ -171,6 +171,22 @@ const signedTransaction = (
   throw new Error(`Error trying to send transaction ${response.data}`)
 }
 
+/**
+ * This endpoint returns the last block stored in the database.
+ * @param {*} db Database
+ * @param {*} Server Config Object
+ */
+const bestBlock = (dbApi: DbApi, { logger }: ServerConfig) => async () => {
+  try {
+    const result = await dbApi.bestBlock()
+    return { Right: { bestBlock: result } }
+  } catch (err) {
```

Comment:
this error handler seems pretty generic, I would expect the error to be handled on an upper level